### PR TITLE
r/aws_sesv2_email_identity_mail_from_attributes: new resource

### PR DESCRIPTION
**Pull request by kamilturek**
### Description
This PR introduces the `aws_sesv2_email_identity_mail_from_attributes` resource.

### Relations
Relates #26796.

### References
https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_PutEmailIdentityMailFromAttributes.html

### Output from Acceptance Testing
```
$ make testacc TESTARGS='-run=TestAccSESV2EmailIdentityMailFromAttributes_' PKG=sesv2 ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/sesv2/... -v -count 1 -parallel 2  -run=TestAccSESV2EmailIdentityMailFromAttributes_ -timeout 180m
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_basic
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_basic
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_disappears
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_disappears
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_basic
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_basic (29.15s)
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure (47.77s)
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_disappears
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_disappears (19.39s)
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain (44.30s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/sesv2      75.237s
```

### CODE REVIEW

File: internal/service/sesv2/email_identity_mail_from_attributes.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"behavior_on_mx_failure": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Default:          string(types.BehaviorOnMxFailureUseDefaultValue),
+				ValidateDiagFunc: enum.Validate[types.BehaviorOnMxFailure](),
```
Comment (kamilturek-sc) [inline internal/service/sesv2/email_identity_mail_from_attributes.go:38]: Wow, this is really handy! Thanks, TIL! 🙌

Comment (kamilturek) [inline internal/service/sesv2/email_identity_mail_from_attributes.go:38]: Wow, this is really handy! Thanks, TIL! 🙌

**Review by jar-b — APPROVED** [review]
LGTM 🎉 

```console
$ make testacc PKG=sesv2 TESTS=TestAccSESV2EmailIdentityMailFromAttributes
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/sesv2/... -v -count 1 -parallel 20 -run='TestAccSESV2EmailIdentityMailFromAttributes'  -timeout 180m
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_basic
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_basic
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_disappears
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_disappears
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_disappearsEmailIdentity
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_disappearsEmailIdentity
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
=== RUN   TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
=== PAUSE TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_basic
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_disappearsEmailIdentity
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain
=== CONT  TestAccSESV2EmailIdentityMailFromAttributes_disappears
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_disappearsEmailIdentity (14.17s)
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_basic (19.28s)
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_disappears (19.63s)
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_mailFromDomain (29.73s)
--- PASS: TestAccSESV2EmailIdentityMailFromAttributes_behaviorOnMXFailure (30.46s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/sesv2      33.155s
```

**Comment by jar-b** [comment]
Thanks for your contribution @kamilturek!

Just made a couple small adjustments:
- replaced `name` attribute references with `email_identity`
- split `_disappears` tests into one where mail from attributes are removed and another where email identity is deleted
- made use of the `enum.Validate` function

**Pull request merged**